crates/zed/src/zed.rs

```diff
@@ -440,14 +443,12 @@ fn initialize_panels(
         };
 
         let (assistant_panel, assistant2_panel) = if is_assistant2_enabled {
-            log::info!("[assistant2-debug] initializing Assistant2");
```

Comment:
I don't think we need to remove these logs.

Reply:
ah, this must have happened when I was trying to get `main` into my branch. I'll revert now.